<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Olivier Trichet &lt;olivier@trichet.fr&gt;</author>
 <class>NewGroupListDatePicker</class>
 <widget class="KDialog" name="NewGroupListDatePicker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>514</height>
   </rect>
  </property>
  <widget class="QWidget" name="mPage" native="true">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>311</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="layout">
    <property name="horizontalSpacing">
     <number>6</number>
    </property>
    <property name="verticalSpacing">
     <number>6</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="1">
     <widget class="KPushButton" name="mLastCheckButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="mLastCheckLabel">
      <property name="text">
       <string notr="true">Date of last check:</string>
      </property>
      <property name="buddy">
       <cstring>mLastCheckButton</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="KDatePicker" name="mDatePicker">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KDialog</class>
   <extends>QDialog</extends>
   <header>kdialog.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KDatePicker</class>
   <extends>QFrame</extends>
   <header>kdatepicker.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
